api/cmd/managerapi.go

```diff
@@ -135,6 +144,7 @@ func NewManagerAPICommand() *cobra.Command {
 	}
 
 	cmd.PersistentFlags().StringVarP(&conf.WorkDir, "work-dir", "p", ".", "current work directory")
+	cmd.PersistentFlags().BoolVarP(&showVersion, "show_version", "v", false, "show manager api-version")
```

Comment:
I think the long option name just can be "version".

Reply:
Got it.